parsl/executors/high_throughput/interchange.py

```diff
@@ -77,6 +77,7 @@ def __init__(self,
                  heartbeat_threshold=60,
                  logdir=".",
                  logging_level=logging.INFO,
+                 poll_period=1,
```

Comment:
I think it should be default to 10 also?

Reply:
You are right. Fixed.